index.html

```diff
@@ -35,7 +35,7 @@
 <body>
 
 <div id="app"></div>
-<script src="generated/js/references-00ad6ccb4d2b9092d23043deedfe0aa81cfe64fe.js"></script>
+<script src="generated/js/references-dd28231060fd294b989f08ab819540d2dce563ad.js"></script>
```

Comment:
Please exclude this change from PR as this section is autogenerated

Reply:
Excluded.